webroot/results/admin/scripts/check_comp_data.php

```diff
@@ -144,7 +144,7 @@
          <ol type='a'>
            <li><a href='check_rounds.php?competitionId=$compIdUrl&show=Show' target='_blank' class='link-external external'>check_rounds</a></li>
            <li><a href='check_regional_record_markers.php?competitionId=$compIdUrl&show=Show' target='_blank' class='link-external external'>check_regional_record_markers</a></li>
-           <li><a href='compute_auxiliary_data.php?doit=+Do+it+now+' target='_blank' class='link-external external'>compute_auxiliary_data</a></li>
+           <li><a href='/admin/compute_auxiliary_data' target='_blank' class='link-external external'>compute_auxiliary_data</a></li>
```

Comment:
Can we get this merged up immediately in a separate PR? I believe this is the link the WRT usually clicks when uploading results. Also, to avoid changing the WRT team's flow, this really should immediately invoke the script, rather than requiring 2 clicks (with a page load inbetween).